abb_irb2400_support/package.xml

```diff
@@ -39,4 +39,18 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>rviz</run_depend>
+
+  <export>
+    <deprecated>
+      The following files/directories will be removed in Lunar:
+      robot_state_visualize_irb2400.launch, use robot_state_visualize_irb2400_12_155.launch || robot_state_visualize_irb2400_20_155.launch
+      load_irb2400.launch, use load_irb2400_12_155.launch || load_irb2400_20_155.launch
+      test_irb2400.launch, use test_irb2400_12_155.launch || test_irb2400_20_155.launch
+      roslaunch_test_2400.xml, use roslaunch_test_2400_12_155.launch || roslaunch_test_2400_20_155.launch
+      irb2400_macro.xacro, use irb2400_12_155_macro.xacro || irb2400_20_155_macro.xacro
+      irb2400.xacro, use irb2400_12_155.xacro || irb2400_20_155.xacro
+      irb2400.urdf, use irb2400_12_155_macro.xacro || irb2400_20_155_macro.xacro
+      meshes/irb2400/, use meshes/irb2400_common/
```

Comment:
I'm confused here: so all the files that you add in this PR will be removed in ROS-L?